server/handler/repo.go

```diff
@@ -7,6 +7,7 @@ import (
 
 	"github.com/drone/drone/plugin/remote"
 	"github.com/drone/drone/server/datastore"
+	"github.com/drone/drone/shared/build/log"
```

Comment:
I realize this is confusing (sorry about that) but the build log only exists for the CLI and is going to be deprecated for something smarter. Let's use the standard `log` package instead